packages/db/fauna/resources/Function/findMetrics.js

```diff
@@ -0,0 +1,82 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Count,
+  Exists,
+  If,
+  Update,
+  Sum,
+  Match,
+  Index,
+  Map,
+  Get,
+  Var,
+  Select,
+  Documents,
+  Collection,
+  Paginate
+} = fauna
+
+const name = 'findMetrics'
+const body = Query(
+  Lambda(
+    [],
+    {
+      data: {
+        usersTotal: Count(Documents(Collection('User'))),
+        uploadsTotal: Count(Documents(Collection('Upload'))),
+        contentTotal: Count(Documents(Collection('Content'))),
+        contentTotalBytes: Sum(Match(Index('content_sizes'))),
+        dealsActiveTotal: Count(Match(
+          Index('deals_by_status'),
+          'Active'
+        )),
+        dealsQueuedTotal: Count(Match(
+          Index('deals_by_status'),
```

Comment:
There are no queued deals - a "queued deal" is the existence of a `Batch` object with no associated `Deal` object.

Reply:
dropping for now